src/cycleclock.h

```diff
@@ -170,7 +170,12 @@ inline BENCHMARK_ALWAYS_INLINE int64_t Now() {
 #elif defined(__s390__)  // Covers both s390 and s390x.
   // Return the CPU clock.
   uint64_t tsc;
+#if defined(BENCHMARK_OS_ZOS) && defined(COMPILER_IBMXL)
+  // z/OS XL compiler HLASM syntax.
+  asm(" stck %0" : "=m"(tsc) : : "cc");
```

Comment:
is the whitespace before stck important? other than that, the only difference is the register constraint as far as i can see. i don't suppose we can replace `Q` with `m` in the non-ZOS branch (still s390)?

Reply:
Yes the whitespace before instruction is intentional as it is part of the inline HLASM syntax and will otherwise cause the compilation error as described in the issue. In terms of constraint I'm using `m` because `Q` isn't supported by the XL compiler on z/OS. I can't comment on whether `Q` can be replaced by `m` on non z/OS branches as I'm only developing on z/OS, so I'm not sure if there will be side effects if we replace `Q` with `m` in the original code below.

Reply:
ok, sounds good. i can't test the other branch but maybe we can replace the whole thing with your new version longer term.